src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumb.jsx

```diff
@@ -42,6 +42,13 @@ class Breadcrumb extends React.Component {
     return [...classes].join(' ');
   };
 
+  getTooltipTitle = () => {
+    const {crumb} = this.props;
+    const parsedTimestamp = moment(crumb.timestamp);
+    const timestampFormat = parsedTimestamp.milliseconds() ? 'll H:mm:ss.SSS A' : 'lll';
```

Comment:
Wondering if we should always be consistent and display ms -- on one hand if the SDK only ever sends in seconds, the formatting would add to clutter, on the other hand if it always sends in ms, it would not display the ms if it's exactly 0. I think the latter is preferable (and just ignore the case where it's exactly 0 ms).

Reply:
@billyvg we should defer to @bruno-garcia on what the payload is like.

Reply:
Even if we changed all SDKs today to send millisecond, it would take years until most apps/sites upgraded etc.
I'd say this looks good.